### PR TITLE
Fixed Dutch translation

### PR DESCRIPTION
**Pull request by moun3m**
Added several translations for untranslated strings. Changed a few mistranslations.

### CODE REVIEW

**Comment by tessus** [comment]
You didn't have to open a new PR. I could have just re-opened the other one. But this PR has a conflict, which has to be resolved.

Also, please have a look at https://github.com/laurent22/joplin/pull/4461#discussion_r572130413

**Comment by tessus** [comment]
Any update?

File: packages/tools/locales/nl_NL.po [hunk 1]
```diff
@@ -3302,9 +3294,8 @@ msgid "Downloaded and encrypted"
 msgstr "Gedownload en versleuteld"
 
 #: packages/lib/services/ReportService.js:180
-#, fuzzy
 msgid "Created locally"
-msgstr "Lokale items gecreëerd: %d."
+msgstr "Lokale items gecreëerd: %d"
```
Comment (tessus) [inline packages/tools/locales/nl_NL.po:3298]: This translation does not seem to be correct. `Lokale items gecreëerd: %d` is already used on line 2988 and is the translation for `Created local items: %d.`.

**Comment by tessus** [comment]
@moun3m Any update? There's still a conflict and the translation on line 3306 is still wrong.

**Comment by laurent22** [comment]
Closing pull request for now. Please let us know if you'd like to continue or feel free to create a new pull request with the changes.

**Pull request closed without merging**